libs/components/storybook/src/lib/preview-wrapper/preview-wrapper-decorators.ts

```diff
@@ -1,13 +1,27 @@
 import { CommonModule } from '@angular/common';
-import { NoopAnimationsModule } from '@angular/platform-browser/animations';
-import { componentWrapperDecorator, moduleMetadata } from '@storybook/angular';
+import { SkyThemeService } from '@skyux/theme';
+import {
+  applicationConfig,
+  componentWrapperDecorator,
+  moduleMetadata,
+} from '@storybook/angular';
 
 import { PreviewWrapperComponent } from './preview-wrapper.component';
 import { PreviewWrapperModule } from './preview-wrapper.module';
 
 export const previewWrapperDecorators = [
   moduleMetadata({
-    imports: [CommonModule, NoopAnimationsModule, PreviewWrapperModule],
+    imports: [CommonModule, PreviewWrapperModule],
+  }),
+  // Define application-wide providers with the applicationConfig decorator
+  applicationConfig({
```

Comment:
Storybook now creates bootstrap applications when generating stories. This tell them what application wide providers should be passed in - see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#angular-application-providers-and-modulewithproviders